doc/dialects/oracle.md

```diff
@@ -170,8 +170,9 @@ The Oracle dialect does not support all capabilities. A complete list can be fou
 
 `NUMBER`, `NUMBER with precision > 36` and `LONG` are casted to `VARCHAR` to prevent a loss of precision. 
 
-If you want to return a DECIMAL type for these types you can set the property ORACLE_CAST_NUMBER_TO_DECIMAL_WITH_PRECISION_AND_SCALE:
-This will cast NUMBER with precision > 36, NUMBER without precision and LONG to DECIMAL(36,20).
+If you want to return a DECIMAL type for these types you can set the property `ORACLE_CAST_NUMBER_TO_DECIMAL_WITH_PRECISION_AND_SCALE` to `<scale>,<precision>`.
```

Comment:
It may not be clear how to do this for newcomers. Can you add an example how to set it? I was thinking as below:

"""
For example,
```
CREATE VIRTUAL SCHEMA <virtual schema name>
    USING ADAPTER.JDBC_ADAPTER
    WITH
    SQL_DIALECT     = 'ORACLE'
    CONNECTION_NAME = 'ORACLE_JDBC_CONNECTION'
    SCHEMA_NAME     = '<schema name>'
    IMPORT_FROM_ORA = 'true'
    ORACLE_CAST_NUMBER_TO_DECIMAL_WITH_PRECISION_AND_SCALE = '2,18'
    ORA_CONNECTION_NAME = 'ORA_CONNECTION';
```
"""